dotnet_task_memory_usage/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"runtime"
+	"strings"
+	"sync/atomic"
+	"time"
+)
+
+func main() {
+	numTasks := flag.Int("numtasks", 1, "number of tasks to run")
+	sleep := flag.Duration("sleep", 1*time.Second, "duration to sleep")
+	flag.Parse() // updates the pointers above to whatever values are provided
```

Comment:
Can you also add a 3rd argument for "max operations" - pollscores will exit the process after that many operations have been counted.  I'll do the same for the c# version.  Pollscores can still sleep every second (hard coded).  We can run the test for a min of 60 seconds so clock time granularity isn't important - measuring CPU used (like you did with /usr/bin/time) is more important.